features-json/colr.json

```diff
@@ -0,0 +1,445 @@
+      "14":"y",
+      "15":"y",
+      "16":"y",
+      "17":"y",
+      "18":"y",
+      "79":"y",
+      "80":"y",
+      "81":"y",
+      "83":"y",
+      "84":"y",
+      "85":"y",
+      "86":"y",
+      "87":"y",
+      "88":"y",
+      "89":"y",
+      "90":"y"
+    },
+    "firefox":{
+      "2":"n",
+      "3":"n",
+      "3.5":"n",
+      "3.6":"n",
+      "4":"n",
+      "5":"n",
+      "6":"n",
+      "7":"n",
+      "8":"n",
+      "9":"n",
+      "10":"n",
+      "11":"n",
+      "12":"n",
+      "13":"n",
+      "14":"n",
+      "15":"n",
+      "16":"n",
+      "17":"n",
+      "18":"n",
+      "19":"n",
+      "20":"n",
+      "21":"n",
+      "22":"n",
+      "23":"n",
+      "24":"n",
+      "25":"n",
+      "26":"y",
+      "27":"y",
+      "28":"y",
+      "29":"y",
+      "30":"y",
+      "31":"y",
+      "32":"y",
+      "33":"y",
+      "34":"y",
+      "35":"y",
+      "36":"y",
+      "37":"y",
+      "38":"y",
+      "39":"y",
+      "40":"y",
+      "41":"y",
+      "42":"y",
+      "43":"y",
+      "44":"y",
+      "45":"y",
+      "46":"y",
+      "47":"y",
+      "48":"y",
+      "49":"y",
+      "50":"y",
+      "51":"y",
+      "52":"y",
+      "53":"y",
+      "54":"y",
+      "55":"y",
+      "56":"y",
+      "57":"y",
+      "58":"y",
+      "59":"y",
+      "60":"y",
+      "61":"y",
+      "62":"y",
+      "63":"y",
+      "64":"y",
+      "65":"y",
+      "66":"y",
+      "67":"y",
+      "68":"y",
+      "69":"y",
+      "70":"y",
+      "71":"y",
+      "72":"y",
+      "73":"y",
+      "74":"y",
+      "75":"y",
+      "76":"y",
+      "77":"y",
+      "78":"y",
+      "79":"y",
+      "80":"y",
+      "81":"y",
+      "82":"y",
+      "83":"y",
+      "84":"y",
+      "85":"y",
+      "86":"y",
+      "87":"y",
+      "88":"y",
+      "89":"y",
+      "90":"y"
+    },
+    "chrome":{
+      "4":"n",
+      "5":"n",
+      "6":"n",
+      "7":"n",
+      "8":"n",
+      "9":"n",
+      "10":"n",
+      "11":"n",
+      "12":"n",
+      "13":"n",
+      "14":"n",
+      "15":"n",
+      "16":"n",
+      "17":"n",
+      "18":"n",
+      "19":"n",
+      "20":"n",
+      "21":"n",
+      "22":"n",
+      "23":"n",
+      "24":"n",
+      "25":"n",
+      "26":"n",
+      "27":"n",
+      "28":"n",
+      "29":"n",
+      "30":"n",
+      "31":"n",
+      "32":"n",
+      "33":"n",
+      "34":"n",
+      "35":"n",
+      "36":"n",
+      "37":"n",
+      "38":"n",
+      "39":"n",
+      "40":"n",
+      "41":"n",
+      "42":"n",
+      "43":"n",
+      "44":"n",
+      "45":"n",
+      "46":"n",
+      "47":"n",
+      "48":"n",
+      "49":"n",
+      "50":"n",
+      "51":"n",
+      "52":"n",
+      "53":"n",
+      "54":"n",
+      "55":"n",
+      "56":"n",
+      "57":"n",
+      "58":"n",
+      "59":"n",
+      "60":"n",
+      "61":"n",
+      "62":"n",
+      "63":"n",
+      "64":"n",
+      "65":"n",
+      "66":"n",
+      "67":"n",
+      "68":"n",
+      "69":"n",
+      "70":"n",
```

Comment:
@drott Is it supported before Chrome 70?

Reply:
In addition, where can I see the version of Opera corresponding to Chrome 70?

Reply:
CFF2, SBIX, CBDT/CBLC are supported [since 66](https://storage.googleapis.com/chromium-find-releases-static/7cd.html#7cd32864f21b540832c3297c68d9473f465e6341), 
COLR/CPAL is universally supported since commit 7a95ad3567708717416e53bbd9d65db9ba2bc4f1, [which means 71](https://storage.googleapis.com/chromium-find-releases-static/7a9.html#7a95ad3567708717416e53bbd9d65db9ba2bc4f1). So marking `"71"` here as `"y"` looks good to me.

See also https://bugs.chromium.org/p/chromium/issues/detail?id=882844

Reply:
> In addition, where can I see the version of Opera corresponding to Chrome 70?

Seems to be Opera 57 according to https://help.opera.com/en/opera-version-history/

Reply:
Thank you so much, I updated Opera's data.

Reply:
> See also https://bugs.chromium.org/p/chromium/issues/detail?id=882844

Yes, I added this bug URL in `notes_by_num 3`.

Reply:
@drott I tested it in Windows 10 and even Chrome 55 supports it. This is rendered by directwrite, right?
Screenshot: https://www.yuque.com/u7268/iconfont/colr-test#UHzkk

Reply:
> @drott I tested it in Windows 10 and even Chrome 55 supports it. This is rendered by directwrite, right?
> Screenshot: https://www.yuque.com/u7268/iconfont/colr-test#UHzkk

Yes, single platform, if the underlying platform rasteriser supports it - so on Windows 10 earlier versions of Chrome supported it. Universal support through FreeType from 71. You could put a foot note for "Windows 10 from 55" but general support was available only from 71.

Reply:
Thanks!